README.md

```diff
@@ -1 +1,222 @@
+
+- [Helios Ransomware Alerts to SecureX and ServiceNow](#get-alerts-to-securex)
+- [Push Helios Ransomware Alerts to Private Intelligence](#push-alerts-to-pi)
+- [Ignore Anomaly on Cohesity Helios](#ignore-anomaly-workflow)
+- [Cohesity Restore Anomalous Object](#restore-anomaly)
+- [Push All Helios Ransomware Alerts to Private Intelligence](#push-all-anomaly)
+
+### <a name="get-alerts-to-securex"></a> Helios Ransomware Alerts to SecureX and ServiceNow 
+[top](#Cisco-SecureX-Integration)
+
+This workflow pushes Cohesity Helios ransomware alerts to Threat Response Private Intelligence data store and also creates incidents on ServiceNow at regular intervals based on the schedule you define. 
+
+This workflow can also be triggered based on a schedule. The schedule that is used here is `Cohesity Helios Ransomware Data Push Schedule` which is a schedule to trigger orchestration workflow to push Helios ransomware data to private intelligence.
+
+##### Input
+
+| **Argument Name** | **Type** | **Description** | **Required** |
+| --- | --- |--- | --- |
+| Threat Response API Client Id | Secure String | Threat Response API Client ID | Yes | 
+| Threat Response API Client Password | Secure String | Threat Response API Client Password | Yes | 
+| Cohesity Helios API Key | Secure String | API Key to access Helios | Yes | 
+| Cohesity Helios Ransomware Alerts Filter  | String | Number of hours, used to get anomalous objects detected in the last `N` hours| Yes | 
+| Create ServiceNow Incidents | String | Set this variable to `yes` or `no` based on which ServiceNow incidents will be created | No | 
+
+##### Output
+
+N/A
+
+### <a name="push-alerts-to-pi"></a> Push Helios Ransomware Alerts to Private Intelligence
+[top](#Cisco-SecureX-Integration)
+
+This workflow pushes Cohesity Helios ransomware alerts detected in the last `N` hours to Threat Response private intelligence data store. 
+
+
+This workflow can also be triggered based on a schedule. The schedule that is used here is `Cohesity Helios Ransomware Data Push Schedule` which is a schedule to trigger orchestration workflow to push Helios ransomware data to private intelligence.
+
+##### Input
+
+| **Argument Name** | **Type** | **Description** | **Required** |
+| --- | --- |--- | --- |
+| Threat Response API Client Id | Secure String | Threat Response API Client ID | Yes | 
+| Threat Response API Client Password | Secure String | Threat Response API Client Password | Yes | 
+| Cohesity Helios API Key | Secure String | API Key to access Helios | Yes | 
+| Cohesity Helios Ransomware Alerts Filter  | String | Number of hours, used to get anomalous objects detected in the last `N` hours| Yes | 
+
+##### Output
+
+N/A
+
+
+### <a name="ignore-anomaly-workflow"></a> Ignore Anomaly on Cohesity Helios
+[top](#Cisco-SecureX-Integration)
+
+This workflow ignores anomaly on for the specified object on Helios which removes this object from the list of anomalous objects. 
```

Comment:
'ignores anomaly on for the' pls remove 'on';
This workflow ignores anomalous object alert on helios of the given object